schemas/cloud.sql

```diff
@@ -74,7 +74,7 @@ CREATE PROCEDURE ReplaceCloudRecord(
   startTime DATETIME, endTime DATETIME,
   suspendDuration BIGINT,
   wallDuration BIGINT, cpuDuration BIGINT,
-  cpuCount INT, networkType VARCHAR(255),  networkInbound BIGINT,
+  cpuCount DECIMAL(10,3), networkType VARCHAR(255),  networkInbound BIGINT,
```

Comment:
I think these should be `UNSIGNED`.

Am I correct in thinking that `(10,3)` will give us 7 leading digits and 3 trailing?